congestion-control-charter.txt

```diff
@@ -58,13 +58,17 @@ both groups.
 
 * Tweaks to existing algorithms, such as Slow Start.
 
+* Techniques to address lower-layer alterations to packet timing or ordering.
+
 * New ways for endpoints to respond to both implicit and explicit congestion
 signals, including specification of explicit signals.
 
 * Progression of existing Informational or Experimental RFCs to higher maturity,
 if they meet the criteria.
 
-* Operational guidance, in general. 
+* Operational guidance, in general, including advice to upper layers regarding
+counterproductive application retransmissions and multiple connections to the
+same host.

```

Comment:
I think it'll be good to clearly mark these as examples.
Are we sure this operational guidance is only to *upper* layers, while we take the lower layers as given? If not, we could generalize this to "other layers" or such.

```suggestion
* Operational guidance, in general, including advice to upper layers such as regarding
counterproductive application retransmissions and multiple connections to the
same host.

```